bin/convert-json-schemas-to-graphql-types.js

```diff
@@ -37,9 +37,11 @@ async function convertDir (dir, asJs) {
 async function run () {
   const argv = yargs
     .boolean('asJs')
+    .boolean('excludeMutations')
+    .usage('Usage: $0 [directory] (--asJs) (--excludeMutations)')
```

Comment:
I thought that `yargs` already added details about the options in the usage. I think that this should be not needed

Reply:
I was somewhat confused, but this could still be made simpler. Can you change it to:

`Usage: $0 <directory> [options]`

With that approach it's clear that the directory is required and then some options can be added